srk/src/linear.ml

```diff
@@ -382,7 +382,7 @@ let max_rowspace_projection a b =
       | None -> ()));
   !c
 
-let max_lds mA mB =
+let max_lds ?zero_rows:(zero_rows=false) mA mB =
```

Comment:
Why do we want to allow zero rows in the simulation?

Reply:
Should I create a separate function for the case `zero_rows=true` for the sake of having a cleaner interface? Right now, I'm somehow misusing max_lds for computing the determined functions for the resets.

Reply:
If you just want the determined functions, you can use the pushout: supposing that `A` is an m-by-n linear map and `X` is a matrix whose rows form a basis for some subspace of Q^n (say of dimension k, so X is a k-by-n matrix), then `pushout A X` returns a pair `(D,C)` which is the "universal solution" to the equation `DA  = CX`.  This means that the rows of `D` form a basis for the space of vectors `d` such that `dA` belongs to the rowspace of X (`dA = cX` for some `c`).

Reply:
I removed this optional argument again, as I'm now using `pushout` instead of `max_lds`.